dynamodb/dynamo_lock_table.go

```diff
@@ -204,3 +215,35 @@ type TableDoesNotExist struct {
 func (err TableDoesNotExist) Error() string {
 	return fmt.Sprintf("Table %s does not exist in DynamoDB! Original error from AWS: %v", err.TableName, err.Underlying)
 }
+
+// Encrypt the TFState Lock table - If Necessary
+func UpdateLockTableSetSSEncryptionOnIfNecessary(tableName string, client *dynamodb.DynamoDB, terragruntOptions *options.TerragruntOptions) error {
+	tableSSEncrypted, err := LockTableCheckSSEncryptionIsOn(tableName, client)
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+
+	if tableSSEncrypted {
+		terragruntOptions.Logger.Printf("Table %s already has encryption enabled", tableName)
+		return nil
+	} else {
+		tableCreateDeleteSemaphore.Acquire()
+		defer tableCreateDeleteSemaphore.Release()
+
+		terragruntOptions.Logger.Printf("Enabling server-side encryption on table %s in AWS DynamoDB", tableName)
+
+		input := &dynamodb.UpdateTableInput{
+			SSESpecification: &dynamodb.SSESpecification{
+				Enabled: aws.Bool(true),
+				SSEType: aws.String("KMS"),
+			},
+			TableName: aws.String(tableName),
+		}
+
+		_, err := client.UpdateTable(input)
+		if err != nil {
+			return errors.WithStackTrace(err)
+		}
+		return errors.WithStackTrace(err)
```

Comment:
Both `return` statements are the same, so can be simplified to just a single `return` in all cases.

Reply:
@brikis98 OK - done.